shared/sharedtest/util.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package sharedtest
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/aetest"
+)
+
+// NewAEInstance creates a new aetest instance backed by dev_appserver whose
+// logs are suppressed. It takes a boolean argument for whether the Datastore
+// emulation should be strongly consistent.
+func NewAEInstance(stronglyConsistentDatastore bool) (aetest.Instance, error) {
```

Comment:
Any reason not to have the datastore always be strongly-consistent?

Reply:
I don't know and based on my very limited experience so far, whenever we need Datastore in tests we always want it to be strongly consistent. What about your experience?

Reply:
IIUC, it's a _little_ slower.

Reply:
Then let's land it as is. With the helper function, we can easily change the default behaviour later on.